index.html

```diff
@@ -0,0 +1,135 @@
+             </label>
+             <input type="email"
+             id="email"
+             placeholder="Enter your email" />
+        </div>
+
+        <div class="form-control">
+           <label for="age" id="age-label">
+               Age
+           </label>
+
+           <input type="text"
+            id="age"
+            placeholder="Enter your age" />
+        </div>
+
+        <div class="form-control">
+           <label for="role" id="role-label">
+                 Which option best describes you?
+           </label>
+
+
+           <select name="role" id="role">
+               <option value="student">Student</option>
+               <option value="intern">Intern</option>
+               <option value="professional">
+                   Professional
+               </option>
+               <option value="other">Other</option>
+               </select>
+          </div>
+
+          <div class="form-control">
+               <label>
+                   Would you recommed Microverse to a friend?
+               </label>
+
+
+               <label for="recommed-1">
+                   <input type="radio"
+                          id="recommed-1"
+                          name="recommed">Yes</input>
+                   </label>
+                   <label for="recommed-2">
+                       <input type="radio"
+                              id="recommed-2"
+                              name="recommed">No</input>
+                   </label>
+                   <label for="recommed-3">
+                   <input type="radio"
+                          id="recommed-3"
+                          name="recommed">Maybe</input>
+                   </label>
+          </div>
+
+          <div class="form-control">
+             <label>Languages and Frameworks known
+                 <small>(Check all that apply)</small>
+             </label>
+
+             <label for="inp-1">
+                 <input type="checkbox"
+                        name="inp">C</input></label>
+             <label for="inp-2">
+                 <input type="checkbox"
+                        name="inp">C++</input></label>
+             <label for="inp-3">
+                 <input type="checkbox"
+                        name="inp">C#</input></label>
+             <label for="inp-4">
+                 <input type="checkbox"
+                        name="inp">Java</input></label>
+             <label for="inp-5">
+                 <input type="checkbox"
+                        name="inp">Python</input></label>
+             <label for="inp-6">
+                 <input type="checkbox"
+                        name="inp">JavaScript</input></label>
+             <label for="inp-7">
+                 <input type="checkbox"
+                        name="inp">React</input></label>
+             <label for="inp-7">
+                 <input type="checkbox"
+                        name="inp">Angular</input></label>
+             <label for="inp-7">
```

Comment:
Your labels for inp-1 to inp-7 don't seem to have any Input element they are referring to

Reply:
You should have phrased this comment in this manner: _Kindly make a reference for your labels for inp-1 to inp-7 as they don't seem to be referencing any input element_